app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/ConfigurationPage.vue

```diff
@@ -164,6 +193,60 @@ export default {
         this.showAlert(this.$t('INBOX_MGMT.EDIT.API.SUCCESS_MESSAGE'));
       }
     },
+    async updateWhatsAppInboxAPIKey() {
+      try {
+        if (this.isAWhatsAppCloudChannel) {
+          const {
+            phone_number_id: phoneNumberId,
+            business_account_id: businessAccountId,
+            webhook_verify_token: webhookVerifyToken,
+          } = this.inbox.provider_config;
+
+          const payload = {
+            id: this.inbox.id,
+            formData: false,
+            provider_config: {
+              api_key: this.whatsAppInboxAPIKey,
+              phone_number_id: phoneNumberId,
+              business_account_id: businessAccountId,
+              webhook_verify_token: webhookVerifyToken,
+            },
+          };
+          await this.$store.dispatch('inboxes/updateInbox', payload);
+        }
+
+        if (this.is360DialogWhatsAppChannel) {
+          const payload = {
+            id: this.inbox.id,
+            formData: false,
+            provider_config: {
+              api_key: this.whatsAppInboxAPIKey,
+            },
+          };
+          await this.$store.dispatch('inboxes/updateInbox', payload);
+        }
+
+        this.showAlert(this.$t('INBOX_MGMT.EDIT.API.SUCCESS_MESSAGE'));
+      } catch (error) {
+        this.showAlert(this.$t('INBOX_MGMT.EDIT.API.SUCCESS_MESSAGE'));
+      }
+    },
   },
 };
 </script>
+<style lang="scss" scoped>
+.whatsapp-settings--content {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+  margin-top: var(--space-small);
+
+  .input {
+    flex: 0.98;
```

Comment:
Why the need for `0.98`?

Reply:
Instead of adding margin-right, just added like this. Want to change?